src/main/java/be/seeseemelk/mockbukkit/services/ServerBuildInfoMock.java

```diff
@@ -0,0 +1,133 @@
+package be.seeseemelk.mockbukkit.services;
+
+import be.seeseemelk.mockbukkit.MockBukkit;
+import io.papermc.paper.ServerBuildInfo;
+import net.kyori.adventure.key.Key;
+import org.jetbrains.annotations.NotNull;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+import java.util.Optional;
+import java.util.OptionalInt;
+import java.util.Properties;
+
+public class ServerBuildInfoMock implements ServerBuildInfo
+{
+
+	private static final Key BRAND_MOCKBUKKIT_ID = Key.key("mockbukkit:mockbukkit");
+	private final String minecraftVersion;
+	private static final String BUILD_DEV = "DEV";
+
+	public ServerBuildInfoMock()
+	{
+		try (InputStream inputStream = MockBukkit.class.getResourceAsStream("/version_info.properties"))
+		{
+			Properties properties = new Properties();
+			properties.load(inputStream);
+			this.minecraftVersion = properties.getProperty("minecraft.version");
+		}
+		catch (IOException e)
+		{
+			throw new RuntimeException(e);
+		}
+	}
```

Comment:
We already have a system for this that doesn't require all this overhead. Look at `BuildParameters.java.peb` and `blossom` block in the `build.gradle.kts`.